vowpalwabbit/cb_explore_adf_rnd.cc

```diff
@@ -166,10 +166,10 @@ float cb_explore_adf_rnd::get_initial_prediction(example* ec)
   std::pair<float, float> dotwithnorm(0.f, 0.f);
   GD::foreach_feature<std::pair<float, float>, const float&, vec_add_with_norm, LazyGaussian>(
     w,
-    all->ignore_some_linear,
-    all->ignore_linear,
-    all->interactions,
-    all->permutations,
+    all->gs.ignore_some_linear,
+    all->fc.ignore_linear,
+    all->gs.interactions,
+    all->fc.permutations,
```

Comment:
When you see two different structures used like this, it brings up interesting questions

Reply:
feels like a halfway between "hey we support turning off and on ignoring features" but not really. Is there any case when we want to have something on the collections (len of list of things to ignore_linear>= 1) and NOT ignore (bool ignore_some_linear= false)? if so, the representation of these structs should represent what is valid and avoid invalid states.